@BrunoQuaresma Do you have an idea why is npm failing here? I can see occurrences like this:

```
e2e/tests/app.spec.ts(6,43): error TS2304: Cannot find name 'beforeCoderTest'.
```

--- a/site/e2e/playwright.config.ts
+++ b/site/e2e/playwright.config.ts
@@ -1,7 +1,6 @@
 import { defineConfig } from "@playwright/test"
 import path from "path"
 import { defaultPort, gitAuth } from "./constants"
-import CoderReporter from "./reporter"
 
 export const port = process.env.CODER_E2E_PORT
   ? Number(process.env.CODER_E2E_PORT)

--- a/site/e2e/tests/app.spec.ts
+++ b/site/e2e/tests/app.spec.ts
@@ -2,6 +2,7 @@ import { test } from "@playwright/test"
 import { randomUUID } from "crypto"
 import * as http from "http"
 import { createTemplate, createWorkspace, startAgent } from "../helpers"
+import { beforeCoderTest } from "../hooks"
 
 test.beforeEach(async ({ page }) => await beforeCoderTest(page))
 

